tests/sliders.test.js

```
@@ -167,6 +170,53 @@ describe('ONNX / VAD', () => {
+  test('app.js _mlCall has a default value of [] for the transfer parameter', () => {
+    expect(appJs).toContain('_mlCall(payload, transfer = [])');
+  });
+
+  test('app.js _mlCall increments an ID counter for request tracking', () => {
+    expect(appJs).toContain('_mlCallId');
+    expect(appJs).toContain('++this._mlCallId');
+  });
+
+  test('app.js _mlCall returns a Promise', () => {
+    // _mlCall wraps responses in a Promise for async/await callers
+    const mlCallBlock = appJs.match(/_mlCall\(payload,[\s\S]*?\n  \}/)?.[0] || '';
```

ESLint

[error] Spaces are hard to count. Use {2}.

(no-regex-spaces)